lib/asset_sync/config.rb

```diff
@@ -94,7 +94,7 @@ def fail_silently?
     end
 
     def log_silently?
-      ENV['RAILS_GROUPS'] == 'assets' || self.log_silently == false
```

Comment:
Should we consider `nil` as `false`?
I know the default value is `true`, but if someone accidentally assigned `nil` to that attribute
Should we throw warning or something?